source/WanManager/wanmgr_dhcpv6_apis.c

```diff
@@ -365,6 +365,7 @@ static int _prepare_client_conf(PDML_DHCPCV6_CFG       pCfg)
         fprintf(fp, "script \"%s\" \n", CLIENT_NOTIFY);
         fprintf(fp, "reconfigure-accept 1\n");
 #else
+        fprintf(fp, "duid-type duid-ll\n");
```

Comment:
It's common so should be moved after the #endif and also be removed from line 364.

Reply:
Done. Thanks.